docs/summits/2025_03/summary.md

```diff
@@ -0,0 +1,498 @@
+# 2025 WheelNext Community Summit - Summary
+
+## **Slides and Resources:** [Link](site:summits/2025_03/slidedecks_and_resources)
+
+## **Key Highlights**
+
+- **Wheel Variants**
+    - Critical for the scientific computing / machine and deep learning ecosystems.
+    - This ecosystem represent a very significant of python users, package downloads, etc.
+- **Index Group Priority** Priority-based resolution to mitigate supply chain risks.
+- **Wheel Evolution** Modernize metadata and compression without breaking users.
+- **Governance** Push for a Packaging Council to unify standards.
```

Comment:
```suggestion
- **Governance** support formation of a Packaging Council, this will improve capacity of the community to review/absorb larger packaging PEPs.
```